docs/account-portal/custom-redirects.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Customizing your redirects | Account Portal
+description:  Customize where your users are redirected to after they sign-in or sign-up.
+---
+
+# Customizing your Account Portal redirects
+
+After a user signs in or signs up, Clerk automatically redirects users back to your application by appending a `redirect_url` query param on the Account Portal page. However, Clerk provides various options for you to customize where your users are redirected to.
+
+## Environment variables
+
+You can define the paths you want the users to be redirected to via environment variables. In this example, we redirect users to `/dashboard` after signing in, and to `/onboarding` after signing up.
+
+<Tabs items={["Next.js", "Remix"]}>
+<Tab>
+```sh filename=".env" copy
+NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/dashboard
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/onboarding
+```
+</Tab>
+<Tab>
+```sh filename=".env" copy
+CLERK_AFTER_SIGN_IN_URL=/dashboard
+CLERK_AFTER_SIGN_UP_URL=/onboarding
+```
+</Tab>
+</Tabs>
```

Comment:
Let's use code block tabs here instead! This is my mistake - I had a different direction and was going to put some content into the Next.js tab but we went a different direction.